hatch.toml

```diff
@@ -40,7 +40,8 @@ extra-dependencies = [
     "pytest-timeout",
 ]
 
-
+[envs.hatch-test.scripts]
+download = "python ./.scripts/ci/download_data.py {args}"
```

Comment:
@selmanozleyen not sure why this wasn't failing before when you merged the hatch PR, I assume you were hitting a cache that expired since then